source/get.d.ts

```diff
@@ -4,13 +4,29 @@ import {Split} from './split';
 /**
 Like the `Get` type but receives an array of strings as a path parameter.
 */
-type GetWithPath<BaseType, Keys extends readonly string[]> =
+type GetWithPath<BaseType, Keys extends readonly string[], Strict extends boolean = false> =
```

Comment:
Might be worth this being a type level "options bag" so it doesn't become a mess if more options are needed

```ts
interface GetOptions {
  strict?: boolean
}

type GetWithPath<BaseType, Keys extends readonly string[], Options extends GetOptions = {}> =
```

Reply:
Yeah good point I agree

Reply:
This is done now :)